src/model_conversion/model_shipper.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2018
+# This notice is to be included in all relevant source files.
+# "Brandon Goldbeck" <bpg@pdx.edu>
+# “Anthony Namba” <anamba@pdx.edu>
+# “Brandon Le” <lebran@pdx.edu>
+# “Ann Peake” <peakean@pdx.edu>
+# “Sohan Tamang” <sohan@pdx.edu>
+# “An Huynh” <an35@pdx.edu>
+# “Theron Anderson” <atheron@pdx.edu>
+# This software is licensed under the MIT License.
+# See LICENSE file for the full text.
+from stl import Mesh
+from src.model_conversion.ldraw_model import LDrawModel
+
+
+class ModelShipper:
+    """The static class responsible for import/export of models.
+
+    Will be mainly responsible for importing STL models and exporting
+    LDraw (.dat) files.
+    """
+
+    @staticmethod
+    def load_stl_model(file_path: str):
+        """Load an STL model and return its mesh data.
+
+        :param file_path: The path to the stl file.
+        :return: The BaseStl model (numpy-stl) loaded from the file_path or None.
+        """
+
+        #TODO: confirm valid filepath
+        return Mesh.from_file(file_path)
+
+    @staticmethod
+    def save_ldraw_file_model(file_path, model: LDrawModel):
```

Comment:
Do we need to close the file `file.close()` at the end by of this method/call another method to do that?